website_product_supplier/views/product_supplier_view.xml

```diff
@@ -0,0 +1,204 @@
+                                                    <label class="control-label">Code</label>
+                                                        <input id="product_code" class="form-control" name="product_code" placeholder="Product Code..."
+                                                               t-att-value="supplierinfo.get('product_code')"/>
+                                                </div>
+                                            </div>
+                                            <div id="supplier_info" class="row">
+                                                <div t-attf-class="col-md-4 col-xs-12 form-group #{error.get('min_qty') and 'has-error' or ''}">
+                                                    <label class="control-label">Min Quantity</label>
+                                                    <div class="controls">
+                                                        <input id="min_qty" class="form-control" name="min_qty" placeholder="Min Quantity..."
+                                                               t-att-value="supplierinfo.get('min_qty')"/>
+                                                    </div>
+                                                </div>
+                                                <div t-attf-class="col-md-4 col-xs-12 form-group #{error.get('delay') and 'has-error' or ''}">
+                                                    <label class="control-label">Delay</label>
+                                                    <div class="controls">
+                                                        <input id="delay" class="form-control" name="delay" placeholder="Delay..."
+                                                               t-att-value="supplierinfo.get('delay')"/>
+                                                    </div>
+                                                </div>
+                                            </div>
+                                            <div id="price_list" class="row">
+                                                <div class="col-md-4 col-xs-12 form-group">
+                                                    <label class="control-label">Price Min Quantity</label>
+                                                    <div class="controls">
+                                                        <input id="min_quantity" class="form-control" name="min_quantity" placeholder="Min Quantity..."
+                                                               t-att-value="pricelist and pricelist[0].min_quantity"/>
+                                                    </div>
+                                                </div>
+                                                <div class="col-md-4 col-xs-12 form-group">
+                                                    <label class="control-label">Price</label>
```

Comment:
It's not clear the difference between Price and Price Min Qty. See comments above.